proxy-node-shared/src/main/java/uk/gov/ida/notification/exceptions/EidasErrorPageException.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ida.notification.exceptions;
+
+import java.util.logging.Level;
+
+public abstract class EidasErrorPageException extends RuntimeException {
```

Comment:
Suggest naming this `ErrorPageException` for consistency with `ErrorPageExceptionMapper`

Reply:
Yep.